test-cargo-miri/Cargo.toml

```diff
@@ -17,6 +17,7 @@ issue_1567 = { path = "issue-1567" }
 issue_1691 = { path = "issue-1691" }
 issue_1705 = { path = "issue-1705" }
 issue_rust_86261 = { path = "issue-rust-86261" }
+executable = { path = "local-crate/executable" }
```

Comment:
This needs comments to explain what the test is about, and ideally a better name -- "executable" doesn't say much. The point is that this one has UB and we're checking what that looks like.

Reply:
I renamed the test `test-local-crate-detection`. Is that better?